UnitTest_Engine/Compute/MergeUnitTests.cs

```diff
@@ -37,10 +37,11 @@ public static partial class Compute
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [PreviousVersion("6.0", "BH.Engine.UnitTest.Compute.MergeUnitTests(System.Collections.Generic.List<BH.oM.Test.UnitTests.UnitTest>)")]
```

Comment:
Unnecessary - this toolkit isn't in beta so versioning isn't required 😄 but good to see you thinking of it at all times/just in case!

Reply:
Yeah, still like to add it to avoid scripts breaking etc. Know it only has effect for people building this, and then building Versioning_Tk, but feel that is better than nothing!